mscore/editstyle.cpp

```diff
@@ -1363,25 +1364,24 @@ void EditStyle::enableStyleWidget(const Sid idx, bool enable)
       }
 
 //---------------------------------------------------------
-//   toggleVerticalJustificationStaves
+//   enableVerticalSpreadClicked
 //---------------------------------------------------------
 
-void EditStyle::toggleVerticalJustificationStaves(bool checked)
+void EditStyle::enableVerticalSpreadClicked(bool checked)
       {
-      enableStyleWidget(Sid::staffDistance, !checked);
-      enableStyleWidget(Sid::akkoladeDistance, !checked);
-      enableStyleWidget(Sid::minSystemDistance, !checked);
-      enableStyleWidget(Sid::maxSystemDistance, !checked);
-
-      enableStyleWidget(Sid::spreadSystem, checked);
-      enableStyleWidget(Sid::spreadSquareBracket, checked);
-      enableStyleWidget(Sid::spreadCurlyBracket, checked);
-      enableStyleWidget(Sid::minSystemSpread, checked);
-      enableStyleWidget(Sid::maxSystemSpread, checked);
-      enableStyleWidget(Sid::minStaffSpread, checked);
-      enableStyleWidget(Sid::maxStaffSpread, checked);
-      enableStyleWidget(Sid::maxAkkoladeDistance, checked);
-      enableStyleWidget(Sid::maxPageFillSpread, checked);
+      disableVerticalSpread->setChecked(!checked);
+      cs->setLayoutAll();
```

Comment:
Not sure why I need to add this sentence to make the score keep up with the change of the group boxes. Without this commit the score will update immediately.